### PR TITLE
Check that created task triggers as expected

### PR DESCRIPTION
**Pull request by mkam**

### CODE REVIEW

File: e2e/condition_catalog_service_test.go [hunk 1]
```diff
@@ -529,6 +529,20 @@ task {
...
+	service = testutil.TestService{ID: "web-test-new", Name: "web-test-new", Tags: []string{"tag_b"}}
+	testutils.RegisterConsulService(t, srv, service, defaultWaitForRegistration)
+
+	// Check that created task does trigger
+	api.WaitForEvent(t, cts, taskName, registrationTime, defaultWaitForEvent)
+	count = eventCount(t, taskName, cts.Port())
+	expectedCount++
+	assert.Equal(t, expectedCount, count)
+	resourcesPath := filepath.Join(tempDir, taskName, resourcesDir)
+	validateServices(t, true, []string{"services_" + sharedService}, resourcesPath)
```
Comment (lornasong) [inline e2e/condition_catalog_service_test.go:543]: I noticed there's a `"services_"` here unlike previous `validateServices()`. I wasn't able to figure out the context for it. Would you mind staring?

**Review by lornasong — APPROVED** [review]
looks good! left a non-blocking question

File: e2e/condition_catalog_service_test.go (hunk 1 above)
Comment (mkam) [reply e2e/condition_catalog_service_test.go:543]: That's due to the module itself adds this in front of the file name for the service! I don't think we check this very often so I can leave a comment here and for the tags file!

Comment (lornasong) [reply e2e/condition_catalog_service_test.go]: Ohhh gotcha. Thank you for explaining that to me. Makes sense.

**Pull request merged**